content/quickstart.md

```diff
@@ -98,5 +105,3 @@ $ watch oc get pods
 ```
 
 You should now be able to open `https://syndesis.$(minishift ip).xip.io` in your browser.
```

Comment:
I recall that [actual PR](https://github.com/openshift/origin/commit/d7d3b7c67dbeea844389bf2d23aa8e4ded6088cc) was meant to go to 3.6 release.